lib/docs/index.html

```diff
@@ -312,8 +307,14 @@
       #searchPlaceholder {
         position: absolute;
         pointer-events: none;
-        top: 5px;
-        left: 5px;
+        height: 100%;
+        display: flex;
+        align-items: center;
+        padding-left: 5px;
```

Comment:
These updated properties help vertically center the placeholder text more reliably.